mconfig

```diff
@@ -117,11 +118,13 @@ usage_args () {
 	echo "     -h   this help"
 	echo
 	echo "  Apptainer options:"
-	echo "     --without-suid    do not install SUID binary (default, linux only)"
-	echo "     --with-suid       do install SUID binary (linux only)"
-	echo "     --without-network do not compile/install network plugins (linux only)"
-	echo "     --without-seccomp do not compile/install seccomp support even if available"
-	echo "     --only-rpm        only configure for rpm (lower host go version check)"
+	echo "     --without-suid     do not install SUID binary (default, linux only)"
+	echo "     --with-suid        do install SUID binary (linux only)"
+	echo "     --without-libsubid do not compile libsubid support (default, linux only)"
```

Comment:
Why not make `--with-subuid` the default if the library is available?

Reply:
> Why not make `--with-subuid` the default if the library is available?

Sounds good, should we just have a `--without-suid` switch then? So by default if we detect libsubid library it will be enabled, unless `--without-suid` set in which case we would ignore.

Reply:
> should we just have a `--without-suid` switch then? So by default if we detect libsubid library it will be enabled, unless `--without-suid` set in which case we would ignore.

That's fine.  It's similar to the `--without-seccomp` case.

Reply:
Ok this should be all set. added `--without-libsubid` switch